### PR TITLE
SeqRegionUtils :  check mapping_set

### PR DESCRIPTION
**Pull request by ima23**
- add check if mapping_set data is not relevant : ENSVAR-3659

### CODE REVIEW

File: modules/t/seqRegionUtils.t [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+BEGIN {
+    use_ok('Bio::EnsEMBL::Variation::Utils::SeqRegionUtils', qw(update_seq_region_ids));
+}
+
+my $multi = Bio::EnsEMBL::Test::MultiTestDB->new('homo_sapiens');
+my $cdb = $multi->get_DBAdaptor('core');
+my $vdb = $multi->get_DBAdaptor('variation');
+
+## check no seq_region update performed when last is not related to current release
```
Comment (helensch) [inline modules/t/seqRegionUtils.t:30]: Suggest comment expand on what last is.
Also might be useful to note that test databases do not have release and assembly on dbname.

**Review by helensch — COMMENTED** [review]
@ima23 thanks for adding additional checks. Decrease in coverage may be due to no test for when mapping_set_id = 0 returned.

**Comment by ima23** [comment]
Thanks for the review @helensch. I've added and expanded the comments. PR is ready for review.

File: modules/t/test-genome-DBs/homo_sapiens/core/mapping_set.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+1	103_38	102_38
```
Comment (helensch) [inline modules/t/test-genome-DBs/homo_sapiens/core/mapping_set.txt:1]: As test dbs are on GRCh37 should this be 103_37,  102_37?

**Review by helensch — APPROVED** [review]

**Comment by helensch** [comment]
merged into master and release/104. 
decrease in coverage was agreed

**Pull request merged**